app/script_variant.py

```diff
@@ -122,7 +156,7 @@ def romanize_word(word):
 
     romanized_words = []
     if has_kanji(word):
-        romanized_words = romanize_japanese_name_by_name_dict(word)
+        romanized_words = romanize_japanese_name_by_name_dict(word, is_create=False)
```

Comment:
You set is_create=False here, which means that you assume this method is used for search queries. But it is not obvious from the method document. Can you do either:

- Add is_create parameter to normalize_word
- Rename the method to romanize_query or something, and document that this method is for search queries.

Note that you should do one of above, not both :)